src/async_fuse/memfs/mod.rs

```diff
@@ -1,5 +1,5 @@
 //! The implementation of user space file system
-mod cache;
+pub mod cache;
```

Comment:
Where is the mod used outside of the memfs module?

Reply:
No usages outside of the `memfs` module for now. But when we refactor the `S3Metadata` with it, the mod `async_fuse` and `async_fuse::test` will need to use this mod, so I need to make it public.

Finally, this mod will be moved to the root of this crate (parallel to mod `async_fuse`), and renamed to `storage`.